src/veh_type.cpp

```diff
@@ -228,8 +233,10 @@ void vpart_info::load_wheel( cata::optional<vpslot_wheel> &whptr, JsonObject &jo
     }
     assign( jo, "rolling_resistance", wh_info.rolling_resistance );
     assign( jo, "contact_area", wh_info.contact_area );
-    wh_info.terrain_mod = standard_terrain_mod;
-    wh_info.or_rating = 0.5f;
+    if( !jo.has_member( "copy-from" ) ) { // if flag presented, it is already set
+        wh_info.terrain_mod = standard_terrain_mod;
+        wh_info.or_rating = 0.5f;
+    }
```

Comment:
Fixed this weirdness when steerable wheels did't have correct `wheel_type`, because they are `copy-from` and it was reset to `standart` here. This also applies to all over wheels.